teamB/kimSungHoon/routes/postRouter.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.post('/register', postController.createPost);
+
+router.get('/allReading', postController.readingPost);
+
+router.patch('/update', postController.updatePost);
+
+router.delete('/delete/:postId', postController.deletePost);
```

Comment:
추후 RESTful API 세션을 통해 배울 내용입니다만 RESTful API에서 엔드포인트를 구성할 땐 동사가 아닌 명사로 작성하게 됩니다. 또한 해당 메서드가 엔드포인트로 드러나지 않게 수정해주세요!

routes/index.js 파일에서 postRouter로 이동할 때의 url이 `/posts`인 것을 고려하여 다음과 같이 작성해줍니다.

엔드포인트 요청 시 url은 다음과 같습니다.
`http://localhost:3000/posts` (get)
`http://localhost:3000/posts/1` (patch)

```suggestion
router.post('/', postController.createPost);
router.get('/', postController.readingPost);
router.patch('/:postId', postController.updatePost);
router.delete('/:postId', postController.deletePost);
```

Reply:
감사합니다 수정완료 했습니다.